spec/factories/pieces.rb

```diff
@@ -16,4 +16,13 @@
     type 'Queen'
     association :game
   end
+
+  factory :rook do
+    color 'white'
+    active true
+    x_position 1
+    y_position 1
```

Comment:
This is why I was asking about how we handle moves that don't actually move. However, since we're including a distance_travelled method, it seems like we wouldn't need to include the extra elements (to_y > y_position & to_y < y_position). 

I'll modify my tests or factory girl shortly.